onadata/apps/api/viewsets/project_viewset.py

```diff
@@ -75,25 +75,24 @@ def get_queryset(self):
 
         return super(ProjectViewSet, self).get_queryset()
 
+    @cache_control(
+        PROJ_CACHE, 60*10, related_cache_entry_prefixes=[PROJ_FORMS_CACHE])
```

Comment:
When I tested this(having a decorator before a viewset action) last time, it seemed the cache_timeout in the decorator was being overridden by the cache timeout set in the CacheControlMixin. Could you please confirm this. How, you may ask?
- Make the cache_timeout in the decorator different with the one in mixin
- Make a get request
- If you are using redis, run`redis-cli` then `ttl <key>`

That said, If we are using the decorator, do we need to use the mixin?

Reply:
Tested this, the timeout is not being overridden by the CacheControlMixin. The mixin and the new decorator have different functions, the `CacheControlMixin` sets the `Cache-Control` header for the responses while the decorator just caches the response. _The naming of the decorator might need to be changed probably..._